src/app/accounts/change-password.component.ts

```diff
@@ -0,0 +1,98 @@
+import {
+    Component,
+} from '@angular/core';
```

Comment:
Could probably one line this one

Reply:
Removed class, fixed in `set-password.component.ts`